client/catapult/scripts/check-license-scope.sh

```
@@ -0,0 +1,77 @@
+CATAPULT_DIR="$(cd "${SCRIPT_DIR}/.." && pwd)" # client/catapult
+cd "${CATAPULT_DIR}"
+
+fail=0
+
+echo "== (1) directory-level LICENSE/COPYING declarations =="
+# Only the two root files are expected; anything else may declare a separate license.
+unexpected="$(find . -type f \( -iname 'LICENSE*' -o -iname 'COPYING*' -o -iname '*.license' \) \
+  | grep -vE '^\./(LICENSE\.txt|COPYING\.LESSER)$' || true)"
+if [ -n "${unexpected}" ]; then
+  echo "  FAIL: unexpected license declaration(s):"
+  printf '    %s\n' ${unexpected}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)